doc/hardware/porting/board_porting.rst

```diff
@@ -106,6 +106,8 @@ the :term:`SoC series` and :term:`SoC family` levels are not always used.
    | :ref:`mimx8mp_evk <imx8mp_evk>`            | mimx8ml8/a53             | i.MX8M Plus | i.MX8M             | NXP i.MX           | Arm Cortex-A53 | ARMv8-A              |
    |                                            +--------------------------+-------------+--------------------+--------------------+----------------+----------------------+
    |                                            | mimx8ml8/m7              | i.MX8M Plus | i.MX8M             | NXP i.MX           | Arm Cortex-M7  | ARMv7-M              |
+   |                                            +--------------------------+-------------+--------------------+--------------------+----------------+----------------------+
```

Comment:
I've proposed this also here: https://github.com/zephyrproject-rtos/zephyr/pull/69802#discussion_r1514425539.
But I got this answer from @tejlmand https://github.com/zephyrproject-rtos/zephyr/pull/69802#discussion_r1514519675
So, maybe you can add info for HiFi4 here also https://docs.zephyrproject.org/latest/boards/nxp/imx8mp_evk/doc/index.html

Reply:
@iuliana-prodan sure will first add dsp info int the document for imx8mp